### ISSUE TITLE
Unicornios

### ISSUE THREAD

**Issue by natxsglez** [dariomtz/taller#2]
Hola, al cuento le hacen falta unos cuantos unicornios... También podría tener arcoíris el cuento.

**Comment by BearCircus** [comment]
Que esta pasando?

**Comment by axelgallardo100900** [comment]
Yo me encargo, dame 5 minutos ;)

**Comment by DasPeter** [comment]
ño

**Comment by BitMightyMike** [comment]
Spider-Man 2 Script - Dialogue Transcript
Voila! Finally, the Spider-Man 2 script is here for all you quotes spouting fans of the movie starring Tobey Maguire and Kirsten Dunst.  This script is a transcript that was painstakingly transcribed using the screenplay and/or viewings of Spider-Man 2. I know, I know, I still need to get the cast names in there and I'll be eternally tweaking it, so if you have any corrections, feel free to drop me a line. You won't hurt my feelings. Honest.
Swing on back to Drew's Script-O-Rama afterwards for more free movie scripts!
Spider-Man 2 Script

   

                   

She looks at me every day.



 

                   

Mary Jane Watson.



 

                   

Oh, boy. If she only knew

how I felt about her.



 

                   

But she can never know.



 

                   

I made a choice once to live

a life of responsibility.



 

                   

A life she can never be a part of.



 

                   

Who am I?



 

                   

I'm Spider-Man, given a job to do.



 

                   

And I'm Peter Parker,

and I, too, have a job.



  

                   

Parker. Parker!



  

                   

No, no, no, stop! Stop!



  

                   

Parker, you're late, man. Always late.



  

                   

I'm sorry, Mr. Aziz.

There was a disturbance.



  

                   

Another disturbance.

Always a disturbance with you.



  

                   

Come on,    minutes ago,

in comes order.



  

                   

Harmattan, Burton & Smith.

Eight extra-large deep-dish pizzas.



  

                   

In eight minutes, I am defaulting on

Joe's   -minute guarantee.



  

                   

Then, not only am I receiving

no money for these pizzas...



  

                   

...but I will lose the customer forever

to Pizza Yurt.



  

                   

Look, you are my only hope, all right?

You have to make it in time.



  

                   

Peter, you're a nice guy.

But you're just not dependable.



  

                   

This is your last chance.



  

                   

You have to go    blocks

in    l  minutes or your ass is fired.



  

                   

Go!



  

                   

Hey, what, are you stupid?



  

                   

Whoa! He stole that guy's pizzas!



  

                   

I'm gonna get it!



  

                   

Hey, you guys.

No playing in the streets.



  

                   

Yes, Mr. Spider-Man.



  

                   

See you.



  

                   

Way to go, Spidey!



  

                   

Pizza time.



  

                   

You're late.



  

                   

I'm not paying for those.



  

                   

Joe's   -minute guarantee

is a promise, man.



  

                   

I know to you, Parker, a promise

means nothing. But to me, it's serious.



  

                   

It's serious to me too, Mr. Aziz.



  

                   

- You're fired. Go.

- Please, I need this job.



  

                   

- You're fired.

- Look, give me another chance.



  

                   

You're fired.



  

                   

- Parker, hello. You're fired.

- Why?



  

                   

- Parker, hello. You're fired.

- Why?



  

                   

Dogs catching Frisbees?



  

                   

Pigeons in the park?

A couple geezers playing chess?



  

                   

- Boss.

- Not now.



  

                   

The Bugle could show another side

of New York for a change.



  

                   

We got six minutes to deadline, Jonah.

We need page one.



  

                   

I don't pay you to be a sensitive artiste.

I pay you... Still not now!



  

                   

I pay you because that psycho

Spider-Man will pose for you.



  

                   

He won't let me take any more pictures.

You turned the whole city against him.



  

                   

A fact I'm very proud of.



  

                   

Get your pretty little portfolio off my

desk before I go into a diabetic coma.



  

                   

- It's your wife, she lost her checkbook.

- Thanks for the good news.



  

                   

Please, isn't there any of these shots

you can use? I need the money.



  

                   

Miss Brant. Get me a violin.



  

                   

Five minutes to deadline, Jonah.



  

                   

Run a picture of a rancid chicken.

Here's the headline:



  

                   

"Food Poisoning Scare Sweeps City."



  

                   

- Some food got poisoned?

- I'm a little nauseous.



  

                   

All right, Mr. Jameson.



  

                   

It stinks.

Robbie, there's your page one.



  

                   

"Masked Menace Terrorizes Town."



  

                   

I told you, he's not a menace.



  

                   

- I told you...

- I'll take care of it.



  

                   

- I'll give you    .

- Three hundred.



  

                   

That's outrageous.



  

                   

Done. Give this to the girl.



  

                   

Thank you. Bye-bye.



  

                   

Hi.



  

                   

Hey, Pete.



  

                   

I don't think this covers the advance

I gave you a couple weeks ago.



  

                   

- Right.

- Sorry.



  

                   

Hey.



  

                   

Chin up, okay?



  

                   

Watch it, jerk!



  

                   

Dr. Connors. Sorry.



  

                   

Where were you headed, Parker?



  

                   

To your class.



  

                   

My class is over.



  

                   

See me standing here?



  

                   

I'm sorry. I'm trying.

I wanna be here.



  

                   

Then be here.



  

                   

Look at you, Peter.



  

                   

Your grades have been steadily

declining. You're late for class.



  

                   

You always appear exhausted.



  

                   

Your paper on fusion is still overdue.



  

                   

I know. I'm planning to write it

on Dr. Otto Octavius.



  

                   

Planning is not a major

at this university.



  

                   

Octavius is a friend of mine.



  

                   

Better do your research, Parker.



  

                   

Get it done, or I'm failing you.



  

                   

Surprise!



  

                   

Well, say something.



  

                   

What's the occasion?



  

                   

Really, Peter. It's your birthday!



  

                   

Whether you want

to remember it or not.



  

                   

He lives in another reality.

Don't you, Pete?



  

                   

- Hi, M.J.

- Hi.



  

                   

- Hey, buddy.

- Hey.



   

                   

Long time, no see.



   

                   

So how's the play?

I read a great review.



   

                   

It's going fine. It's going good.



   

                   

She's brilliant in it.



   

                   

Harry sent me roses.



   

                   

So where you been, pal?

You don't return my calls.



   

                   

I've been busy.



   

                   

Taking pictures of Spider-Man?

How's the bug these days?



   

                   

The less you see of that man,

the better.



   

                   

Now, let's all go into the other room

and have something to eat.



   

                   

I'll get the hors d'oeuvres.



   

                   

So how are things going at Oscorp?



   

                   

They're great.

I'm head of Special Projects.



   

                   

We're about to make

a breakthrough on fusion.



   

                   

How lovely, Harry. Your father

would be so proud, rest his soul.



   

                   

Thank you.



   

                   

We're actually funding

one of your idols, Pete. Otto Octavius.



   

                   

- I'm writing a paper on him.

- You want to meet him?



   

                   

You'd introduce me?



   

                   

You bet. Octavius is

gonna put Oscorp on the map...



   

                   

...in a way my father

never even dreamed of.



   

                   

M.J., could you give me a hand?



   

                   

She's waiting for you, pal.



   

                   

What do you mean?



   

                   

The way she looks at you

or doesn't look at you.



   

                   

However you want to look at it.



   

                   

I don't have time for girls right now.



   

                   

Why, are you dead?



   

                   

I've been kind of busy.



   

                   

Taking pictures of your friend?



   

                   

Could we get off that subject?



   

                   

I want us to be friends, Harry.

I want us to trust each other.



   

                   

Then be honest with me.



   

                   

If you knew who he was,

would you tell me?



   

                   

May.



   

                   

Oh, what, Ben?



   

                   

Wait.



   

                   

- Aunt May.

- Oh, my.



   

                   

Oh, Peter. Oh, for a second there,

I thought I was years ago.



   

                   

Everybody's gone, aren't they?



   

                   

Did they have a good time?



   

                   

I'm sure they did.



   

                   

- You okay?

- Of course. But you go home.



   

                   

And be careful. I don't like that

scooter thing you drive around.



   

                   

I'm worried about you.



   

                   

You're so alone.

And I saw the letter from the bank.



   

                   

Oh, my. You did?



   

                   

Oh, well.



   

                   

So?



   

                   

I'm a little behind.



   

                   

Everybody is.



   

                   

Anyway, I don't want

to talk about it anymore.



   

                   

I'm tired,

and you better start back home.



   

                   

Here, kiddo. Happy birthday.



   

                   

You need it more than I do.



   

                   

- No, I can't take that from you.

- Yes, you can!



   

                   

You can take this money from me.



   

                   

For God's sake, it's not much.

Now, take it!



   

                   

And don't you dare leave it here.



   

                   

Oh, I'm sorry.



   

                   

It's just that I miss

your uncle Ben so much.



   

                   

Can you believe that it's two years

next month since he was taken?



   

                   

I think to myself at times...



   

                   

...were I to face the one responsible

for what happened, I'd...



   

                   

Oh, I don't know what I'd do.



   

                   

Now...



   

                   

...you better take

the rest of your cake home.



   

                   

Hey.



   

                   

Hey. You're still here.



   

                   

I saw your billboard on Bleecker.



   

                   

Isn't it funny?

I'm really kind of embarrassed.



   

                   

Don't be. It's nice.

I get to see you every day now.



   

                   

I liked seeing you tonight, Peter.



   

                   

Oh, boy, yeah.



   

                   

"Oh, boy, yeah" what?



   

                   

Nothing.



   

                   

Do you want to say something?



   

                   

I...



   

                   

...was...



   

                   

...wondering if you're still

in the Village.



   

                   

You're such a mystery.



   

                   

Peter.



   

                   

What?



   

                   

Happy birthday.



   

                   

I'm seeing somebody now.



   

                   

You mean, like a boyfriend?



   

                   

Well, like I like him.



   

                   

What?



   

                   

Nothing.



   

                   

That's good, you know?

Companionship...



   

                   

May be more than that.



   

                   

More?



   

                   

I don't know.



   

                   

I'm coming to see your play

tomorrow night.



   

                   

- You're coming?

- I'll be there.



   

                   

Don't disappoint me.



   

                   

I won't.



   

                   

Rent.



   

                   

- Hi.

- Hi. What's "hi"?



   

                   

Can I spend it?



   

                   

I have a paycheck due this week

and...



   

                   

- You're a month late again. Again.

- I promise as soon as...



   

                   

If promises were crackers,

my daughter would be fat.



   

                   

I'm really sorry, Mr. Ditkovitch. All I got

is this    for the rest of the week.



   

                   

"Sorry" doesn't pay the rent.



   

                   

And don't try to sneak past me. I have

ears like a cat and eyes like a rodent.



   

                   

- Thanks, Mr. Ditkovitch.

- Hi, Pete.



   

                   

Hello?



   

                   

Rent?



   

                   

Doctor? Mr. Osborn's here.



   

                   

Nobel Prize, Otto. Nobel Prize.

And we'll all be rich.



   

                   

It's not about the prizes, Harry.



   

                   

But you need money.

You need Oscorp.



   

                   

Who do we have here?



   

                   

This is my good friend

I called you about.



   

                   

He got me through

high-school science.



   

                   

Peter Parker, sir.

I'm writing a paper on you for...



   

                   

I know what you're doing here...



   

                   

...but I don't have time

to talk to students now.



   

                   

But Oscorp pays the bills, so...



   

                   

That's why I have to take off.

Board meeting.



   

                   

But my job is done here.

Got you two geniuses together.



   

                   

Good luck tomorrow, Otto.

Nobel Prize. We'll see you in Sweden!



   

                   

Interesting fellow, your friend.



   

                   

I won't take much of your time.



   

                   

Now I remember you. You're Connors'

student. He tells me you're brilliant.



   

                   

He also tells me you're lazy.



   

                   

I'm trying to do better.



   

                   

Being brilliant's not enough,

young man. You have to work hard.



   

                   

Intelligence is not a privilege,

it's a gift.



   

                   

And you use it

for the good of mankind.



   

                   

You want to try it back there?



   

                   

So is that it?



   

                   

Yes. My design to initiate

and sustain fusion.



   

                   

I understand you use harmonics

of atomic frequencies.



   

                   

Sympathetic frequencies.



   

                   

Harmonic reinforcement?



   

                   

Go on.



   

                   

An exponential increase

in energy output.



   

                   

A huge amount of energy.



   

                   

Like a perpetual sun providing

renewable power for the whole world.



   

                   

Are you sure you could stabilize

the fusion reaction?



   

                   

Peter, what have we been talking

about for the last hour and a half?



   

                   

This is my life's work.



   

                   

I certainly know the consequences

of the slightest miscalculation.



   

                   

I'm sorry. I didn't mean

to question you.



   

                   

Rosie, our new friend thinks

I'm gonna blow up the city.



   

                   

You can sleep soundly tonight.



   

                   

Otto's done his homework.



   

                   

Come to the demonstration tomorrow,

and you'll see for yourself.



   

                   

And you need

to sleep soundly tonight, Otto.



   

                   

Did Edison sleep

before he turned on the light?



   

                   

Did Marconi sleep

before he turned on the radio?



   

                   

Did Beethoven sleep

before he wrote the  th?



   

                   

Did Bernoulli sleep before he found

the curves of quickest descent?



   

                   

Rosie, I love this boy.



   

                   

Peter, tell us about yourself.

Do you have a girlfriend?



   

                   

Well...



   

                   

I don't really know.



   

                   

Well, shouldn't you know?

I mean, who would know?



   

                   

Leave him alone.

Maybe it's a secret love.



   

                   

Love should never be a secret.



   

                   

If you keep something as complicated

as love stored up inside...



   

                   

...gonna make you sick.



   

                   

I finally got lucky in love.



   

                   

We both did.



   

                   

But it's hardly perfect.

You have to work at it.



   

                   

I met him on the college steps,

and I knew it wasn't going to be easy.



   

                   

He was studying science,

and I was studying English literature.



   

                   

That's right. I was trying to explain

the theory of relativity.



   

                   

And Rosie was trying

to explain T.S. Eliot.



   

                   

I still don't understand

what he was talking about.



   

                   

- Yes, you do.

- I'm serious.



   

                   

T.S. Eliot is more complicated

than advanced science.



   

                   

But if you want to get a woman to fall

in love with you, feed her poetry.



   

                   

Poetry.



   

                   

Never fails.



   

                   

A tall and slender maiden

All alone upon a prairie



   

                   

Brightest green were all her garments



   

                   

And her hair was like the sunshine



   

                   

Day by day he gazed upon her



   

                   

Rent!



   

                   

Where is my money?!



   

                   

You seem jittery tonight.



   

                   

You never know who's coming.



   

                   

Ladies, five minutes. Five minutes.



   

                   

How'd you do that?



   

                   

Work out, plenty of rest.



   

                   

You know, eat your green vegetables.



   

                   

That's what my mom is always saying.

I just never actually believed her.



   

                   

Come on. Go.



   

                   

Keep it steady.



   

                   

It's a web.



   

                   

Go, Spidey, go!



   

                   

We got trouble!



   

                   

Get me down!



   

                   

I am more than content

with what Mr. Moncrieff said.



   

                   

His voice alone inspires one

with absolute credulity.



   

                   

Then you think we should

forgive them?



   

                   

Yes.



   

                   

I mean, no.



   

                   

Hey. Hey. Hey, chief.

You park there, I'm towing it.



   

                   

Whatever.



   

                   

Shoelace.



   

                   

You might want to...



   

                   

- Can I help you?

- Yeah, I've come to see the show.



   

                   

Oh, I'm sorry, sir. No one will be

seated after the doors are closed.



   

                   

It helps maintain the illusion.



   

                   

Miss Watson, she's a friend of mine.

She asked me to come.



   

                   

But not to come late.



   

                   

I have to see this show.

Just let me in, I'll stand in the...



   

                   

A Spider-Man

A Spider-Man



   

                   

A-does whatever a spider can



   

                   

A-spins a web, any size



   

                   

Catches thieves a-just like flies



   

                   

Look out

Here comes the Spider-Man



   

                   

You sure you don't wanna

come tonight?



   

                   

Yeah, I'm sure.



   

                   

- Okay. Call me.

- I will.



   

                   

- You were great tonight.

- Thanks.



   

                   

May I have an autograph,

please, miss?



   

                   

What are you doing here?



   

                   

You hungry?



   

                   

Starved.



   

                   

What was that?



   

                   

Okay.



   

                   

Cool Spidey outfit.



   

                   

Thanks.



   

                   

Where'd you get it?



   

                   

I made it.



   

                   

Looks uncomfortable.



   

                   

Yeah, it gets kind of itchy.



   

                   

And it rides up in the crotch

a little bit too.



   

                   

Hi. It's me.

Sing your song at the beep.



   

                   

Hi, M.J.



   

                   

This is Peter.



   

                   

I was on my way to your show and...



   

                   

Well, I was on my bike...



   

                   

Are you there?



   

                   

I really was planning on it all day.



   

                   

And...



   

                   

...I know you predicted

I'd disappoint you.



   

                   

Bingo.



   

                   

It's amazing, isn't it?



   

                   

How complicated a simple thing...



   

                   

...like being someplace at  :  

can become.



   

                   

Actually, there was

this obnoxious usher.



   

                   

Somebody has to talk

to that usher, M.J.



   

                   

Your time has expired.



   

                   

Please deposit    cents

for the next five minutes.



   

                   

I wanna tell you the truth.



   

                   

Here it is.



   

                   

I'm Spider-Man.



   

                   

Weird, huh?



   

                   

Now you know

why I can't be with you.



   

                   

If my enemies found out about you...



   

                   

...if you got hurt,

I could never forgive myself.



   

                   

I wish I could tell you

how I feel about you.



   

                   

Ladies and gentlemen...



   

                   

...my wife Rosie and I would like to

welcome you this afternoon.



   

                   

But first, before we start...



   

                   

...has anybody lost a large roll

of   -dollar bills in a rubber band?



   

                   

Because we found the rubber band.



   

                   

It's a terrible joke.



   

                   

But thank you for coming.



   

                   

Today...



   

                   

...you will witness the birth

of a new fusion-based energy source.



   

                   

Safe, renewable energy

and cheap electricity for everyone.



   

                   

And now let me introduce

my assistants.



   

                   

These four actuators were

developed and programmed...



   

                   

...for the sole purpose of creating

successful fusion.



   

                   

They are impervious to heat

and magnetism.



   

                   

These smart arms are controlled

by my brain through a neural link.



   

                   

Nanowires feed directly

into my cerebellum...



   

                   

...allowing me to use these arms

to control fusion reaction...



   

                   

...in an environment

no human hand could enter.



   

                   

Doctor, if the artificial intelligence

in the arms...



   

                   

...is as advanced as you suggest...



   

                   

...couldn't that make you

vulnerable to them?



   

                   

How right you are.



   

                   

Which is why I developed

this inhibitor chip...



   

                   

...to protect

my higher brain function.



   

                   

It means I maintain control of these

arms, instead of them controlling me.



   

                   

And now on to the main event.



   

                   

Give me the blue light, Rosie.



   

                   

Precious tritium is the fuel

that makes this project go.



   

                   

There's only    pounds of it

on the whole planet.



   

                   

I'd like to thank Harry Osborn and

Oscorp Industries for providing it.



   

                   

Happy to pay the bills, Otto.



   

                   

Ladies and gentlemen...



   

                   

...fasten your seat belts.



   

                   

Doctor, we have

a successful fusion reaction.



   

                   

This is a breakthrough

beyond your father's dreams.



   

                   

Thank you.



   

                   

We're producing

a thousand-megawatt surplus.



   

                   

The power of the sun...



   

                   

...in the palm of my hand.



   

                   

Keep calm! It's only a spike!



   

                   

lt'll soon stabilize!



   

                   

Ladies and gentlemen,

please clear the room.



   

                   

We have a containment breach!



   

                   

Otto, please get back!



   

                   

Shut it off, Otto! Shut it off!



   

                   

It will stabilize! It's under control!



   

                   

I'm in charge here!



   

                   

It's my money! I'm in con...



   

                   

This doesn't change anything.



   

                   

- What are you doing?

- Pulling the plug.



   

                   

No!



   

                   

Rosie!



   

                   

Watch it.



   

                   

That was too close.



   

                   

If he had more than a drop of tritium,

he could have destroyed the city.



   

                   

I'm ruined.



   

                   

I have nothing left,

except Spider-Man.



   

                   

He saved your life, sir.



   

                   

He humiliated me by touching me.



   

                   

The press will be here soon.

I suggest we move on.



   

                   

What was he doing here anyway?



   

                   

Molten metal penetrated

the spinal cavity...



   

                   

...and fused the vertebrae

at multiple points...



   

                   

...including the lamina

and the roof of the spinal column.



   

                   

We won't know the extent of

the damage until we get in there.



   

                   

I suggest we cut off these mechanical

arms, slice up the harness...



   

                   

...and, if need be,

consider a laminectomy...



   

                   

...with posterior spinal fusion

from C  -T  to T  .



   

                   

We're ready, doctor.



   

                   

Anybody here take shop class?



   

                   

Help me! Help me!



   

                   

No!



   

                   

It's all over town, Robbie.

Gossip. Rumors.



   

                   

Panic in the streets, if we're lucky.



   

                   

Crazy scientist turns himself

into some kind of a monster.



   

                   

Four mechanical arms welded

right onto his body.



   

                   

Guy named Otto Octavius winds up

with eight limbs. What are the odds?



   

                   

Hoffman!



   

                   

What are we gonna call this guy?



   

                   

- "Dr. Octopus."

- That's crap.



   

                   

- "Science Squid"?

- Crap!



   

                   

- "Dr. Strange."

- That's pretty good.



   

                   

But it's taken.

Wait, wait! I got it. "Dr. Octopus."



   

                   

- I like it.

- Of course you do.



   

                   

Dr. Octopus. New villain in town.

"Doc Ock."



   

                   

- Genius.

- What, are you looking for a raise?



   

                   

Get out.



   

                   

Chief, I found Parker.



   

                   

Where you been?

Why don't you pay your phone bill.



   

                   

Mad scientist goes berserk,

we don't have pictures!



   

                   

I heard Spider-Man was there.



   

                   

Where were you?

Photographing squirrels? You're fired.



   

                   

Chief, the planetarium party.



   

                   

You're un-fired.

I need you, come here.



   

                   

What do you know

about high society?



   

                   

- Oh. Well, l...

- Don't answer that.



   

                   

My society photographer got hit in the

head by a polo ball. You're all I got.



   

                   

Big party for an American hero.

My son, the astronaut.



   

                   

Could you pay me in advance?



   

                   

You serious? Pay you for what?

Standing there?



   

                   

The planetarium, tomorrow night,

 :  . There's the door.



   

                   

My Rosie's dead.



   

                   

My dream is dead.



   

                   

And these...



   

                   

...monstrous things should be

at the bottom of the river...



   

                   

...along with me.



   

                   

Something...



   

                   

...in my head.



   

                   

Something talking.



   

                   

The inhibitor chip!



   

                   

Gone.



   

                   

Rebuild.



   

                   

No.



   

                   

Peter was right.



   

                   

I miscalculated.



   

                   

I couldn't have miscalculated.



   

                   

It was working, wasn't it?



   

                   

Yes.



   

                   

We could rebuild.

Enlarge the containment field.



   

                   

Make it bigger

and stronger than ever.



   

                   

But we need money.



   

                   

Steal it?



   

                   

No, no, no, I'm not a criminal.



   

                   

That's right.



   

                   

The real crime would be not to finish

what we started.



   

                   

We'll do it here.



   

                   

The power of the sun

in the palm of my hand.



   

                   

Nothing will stand in our way.



   

                   

Nothing!



   

                   

- That's the social security.

- Yes, I see.



   

                   

My uncle Ben's life insurance.



   

                   

Yes, but I'm afraid...



   

                   

...it's just not sufficient

to refinance your home.



   

                   

Oh, but I'm giving

piano lessons again.



   

                   

You are?



   

                   

We appreciate

that you've just opened up...



   

                   

...a new supersaver account

with us today.



   

                   

But the fact is...



   

                   

...you do not have the assets

to justify this loan. I'm sorry.



   

                   

Well...



   

                   

At least we get the toaster.



   

                   

Actually, that's only

with a deposit of     or more.



   

                   

Oh, yes, I see.



   

                   

Okay.



   

                   

Don't worry, we'll figure it out.



   

                   

Peter?



   

                   

Don't leave me.



   

                   

Oh, that boy of yours is a real hero.



   

                   

Hold it!



   

                   

Put your arms up! All of them!



   

                   

Here's your change!



   

                   

Oh, no. Come on!



   

                   

- You're getting on my nerves.

- I have a knack for that.



   

                   

Not anymore.



   

                   

- Freeze! Don't move!

- Freeze!



   

                   

- Hold your fire!

- Don't follow me.



   

                   

Wait a second.



   

                   

- Hand her over.

- Of course.



   

                   

Easy, now.



   

                   

Butterfingers.



   

                   

Look out!



   

                   

Help me!



   

                   

- Help me!

- I'm coming!



   

                   

Hang on!



   

                   

Help!



   

                   

Thank you.



   

                   

Aunt May.



   

                   

You've stuck your webs

in my business for the last time.



   

                   

Now you'll have this woman's death

on your conscience.



   

                   

Come on.



   

                   

Shame on you.



   

                   

There you go.



   

                   

Thank you.

Oh, have I been wrong about you.



   

                   

We sure showed him.



   

                   

What do you mean, "we"?



   

                   

- Take me with you, Spider-Man.

- Take me.



   

                   

- May I?

- Sure. Go ahead.



   

                   

Penny! Over here!



   

                   

Leave it.



   

                   

Hey. Might wanna take it easy, buddy.



   

                   

Why? It's a party.



   

                   

Wouldn't you be drinking if you lost

a bundle on some crackpot...



   

                   

...who you thought was gonna take

you with him to fame and fortune?



   

                   

Not to mention your friend the bug.



   

                   

Not tonight, Harry.



   

                   

Every night!

Until I find him, it's   - .



   

                   

Parker!



   

                   

Parker!



   

                   

Parker! What, are you deaf?

I called you twice.



   

                   

Think I'm paying you

to sip champagne?



   

                   

Get a shot of my wife

with the minister.



   

                   

Beautiful tie.



   

                   

Get a shot of us with the DA.



   

                   

Beautiful dress.



   

                   

Here, get a shot of the mayor

and his girlfriend.



   

                   

Wife.



   

                   

Ladies and gentlemen,

good evening.



   

                   

The committee for

the Science Library of New York...



   

                   

...is pleased to present

our guest of honor.



   

                   

He's the first man to play football

on the moon.



   

                   

The handsome, the heroic,

the delicious...



   

                   

...Captain John Jameson.



   

                   

Hi.



   

                   

- You.

- Listen, I'm sorry.



   

                   

But there was a disturbance.



   

                   

I don't know you.



   

                   

And I can't keep thinking about you.

It's too painful.



   

                   

I've been reading poetry lately.



   

                   

Whatever that means.



   

                   

Day by day he gazed upon her



   

                   

Day by day he sighed with passion

Day by day...



   

                   

Don't start.



   

                   

Can I get you a drink?



   

                   

I'm with John. He'll get me my drink.



   

                   

John.



   

                   

By the way, John has seen

my show five times.



   

                   

Harry has seen it twice.

Aunt May has seen it.



   

                   

My sick mother got out of bed to see it.



   

                   

Even my father.



   

                   

He came backstage to borrow cash.



   

                   

But my best friend,

who cares so much about me...



   

                   

...can't make an  :   curtain.



   

                   

After all these years...



   

                   

...he's nothing to me

but an empty seat.



   

                   

It pisses me off, your loyalty to

Spider-Man and not to your best friend.



   

                   

I find him with my father's body,

and you defend him.



   

                   

- Take it easy.

- Don't push me.



   

                   

Don't act like you're my friend.



   

                   

You stole M.J. from me.



   

                   

You stole my father's love.



   

                   

Then you let him die because you

didn't turn in the freak. Isn't that right?



   

                   

Huh? Isn't that right?



   

                   

Huh, brother?



   

                   

Ladies and gentlemen,

I just want you all to know...



   

                   

...that the beautiful Miss Mary Jane

Watson has just agreed to marry me.



   

                   

What a surprise.



   

                   

Parker, wake up! Wake up!

Shoot the picture!



   

                   

Oh, no. No!



   

                   

Why is this happening to me?



   

                   

You seem very okay to me.



   

                   

My diagnosis?



   

                   

It's up here.



   

                   

You say you can't sleep.

Heartbreak? Bad dreams?



   

                   

There is one dream where...



   

                   

...in my dream...



   

                   

...l'm Spider-Man.



   

                   

But I'm losing my powers.

I'm climbing a wall, but I keep falling.



   

                   

So you're Spider-Man.



   

                   

In my dream.



   

                   

Actually, it's not even my dream.

It's a friend of mine's dream.



   

                   

Somebody else's dream.



   

                   

What about this friend?



   

                   

Why does he climb these walls?

What does he think of himself?



   

                   

That's the problem.

He doesn't know what to think.



   

                   

Gotta make you mad not to know

who you are. Your soul disappears.



   

                   

Nothing as bad as uncertainty.



   

                   

Listen...



   

                   

...maybe you're not supposed to be

Spider-Man climbing those walls.



   

                   

That's why you keep falling.



   

                   

You always have a choice, Peter.



   

                   

I have a choice.



   

                   

All the things you've

been thinking about, Peter...



   

                   

...make me sad.



   

                   

Can't you understand?



   

                   

I'm in love with Mary Jane.



   

                   

Peter, all the times

we've talked of honesty...



   

                   

...fairness, justice...



   

                   

Out of those times,

I counted on you to have the courage...



   

                   

...to take those dreams

out into the world.



   

                   

I can't live your dreams anymore.



   

                   

I want a life of my own.



   

                   

You've been given a gift, Peter.



   

                   

With great power

comes great responsibility.



   

                   

Take my hand, son.



   

                   

No, Uncle Ben.



   

                   

I'm just Peter Parker.



   

                   

I'm Spider-Man...



   

                   

...no more.



   

                   

No more.



   

                   

No more.



   

                   

- You punk!

- Sorry.



   

                   

- Thanks.

- You're welcome.



   

                   

Hurry up!



   

                   

And when the borderline is tau

equals zero, the eigenvalues are...?



   

                   

Someone, please?



   

                   

.   electron volts.



   

                   

Excellent work today, Parker.

Keep it up.



   

                   

I'm not really wicked at all.

You mustn't think that I'm wicked.



   

                   

I'm not really wicked at all.

You mustn't think that I'm wicked.



   

                   

If you are not, then you have

certainly been deceiving us all...



   

                   

...in a very inexcusable manner.



   

                   

I hope you have not been

leading a double life...



   

                   

...pretending to be wicked

and being really good all the time.



   

                   

That would be hypocrisy.



   

                   

Of course, I have been

rather reckless.



   

                   

"I am glad."



   

                   

"I am glad."



   

                   

"I am glad."



   

                   

Are you glad to hear it?



   

                   

I am glad...



   

                   

...to hear it.



   

                   

In fact, now you mention

the subject...



   

                   

...I have been very bad

in my own small way.



   

                   

You were so wonderful.

That was such a great play.



   

                   

You could've told me

you were coming.



   

                   

I was afraid you'd say,

"Don't come."



   

                   

You look different.



   

                   

I shined my shoes, pressed my pants,

did my homework.



   

                   

I do my homework now.



   

                   

You wanna get some chow mein?



   

                   

Peter...



   

                   

...l'm getting married.



   

                   

I always imagined you

getting married on a hilltop.



   

                   

And who's the groom?



   

                   

You hadn't decided yet.



   

                   

You think just because

you saw my play...



   

                   

...you can talk me out

of getting married?



   

                   

You once told me you loved me.

I let things get in the way before.



   

                   

There was something I thought

I had to do. I don't have to.



   

                   

You're too late.



   

                   

Will you think about it?



   

                   

Think about what?



   

                   

Picking up where we left off.



   

                   

Where was that?



   

                   

We never got on.

You can't get off if you don't get on.



   

                   

I don't think it's that simple.



   

                   

Of course you don't,

because you complicate things.



   

                   

You don't understand.



   

                   

I'm not an empty seat anymore.



   

                   

I'm different.



   

                   

Punch me, I bleed.



   

                   

I have to go.



   

                   

I'm getting married in a church.



   

                   

You are different.



   

                   

Dear, we agreed to put on a wedding,

not go into bankruptcy.



   

                   

Caviar?

What, are we inviting the czar?



   

                   

Get some cheese and crackers,

some of those little cocktail weenies.



   

                   

I got a garbage man here says he's

got something you might want to see.



   

                   

If you have an extraterrestrial's head in

there, you're the third guy this week.



   

                   

Where the hell did you get that?



   

                   

In the garbage.



   

                   

In the garbage?



   

                   

He must've given up.



   

                   

Thrown in the towel.

Abandoned his sad little masquerade.



   

                   

I finally got to him.

The power of the press triumphs.



   

                   

Yeah, yeah, yeah. Now, look, I think

I deserve a little something for this.



   

                   

- Give you    bucks.

- I could get more than that on eBay.



   

                   

All right,    . Give this man his

money and throw in a bar of soap.



   

                   

Your wife is still on that line.



   

                   

Flowers? How much?



   

                   

You spend any more on this thing,

you can pick the daisies off my grave.



   

                   

Get plastic.



   

                   

Gotta go, big story.



   

                   

Spider-Man

Spider-Man



   

                   

Where have you gone to

Spider-Man?



   

                   

Where have you gone to

Spider-Man?



   

                   

Come on, man. Get the money.



   

                   

Come on, come on.



   

                   

Help!



   

                   

Come on!



   

                   

It wasn't fair to have gone like that.



   

                   

He was a peaceful man.



   

                   

And it was all my doing.



   

                   

Aunt May.

You don't have to punish yourself.



   

                   

Oh, I know I shouldn't.



   

                   

It's just that you wanted

to take the subway...



   

                   

...and he wanted to drive you.



   

                   

If only I had stopped him.



   

                   

We'd all three of us

be having tea together.



   

                   

I'm responsible.



   

                   

For what?



   

                   

For what happened to Uncle Ben.



   

                   

But you were at the library.



   

                   

You were doing your homework.



   

                   

He drove me to the library,

but I never went in.



   

                   

What do you mean?



   

                   

I went someplace else.



   

                   

Someplace where I thought

I could win some money...



   

                   

...to buy a car, because I wanted...



   

                   

...to impress Mary Jane.



   

                   

It happened so fast.



   

                   

I won the money, and the guy

wouldn't pay me. Then he got robbed.



   

                   

The thief...



   

                   

...was running towards me.



   

                   

I could've stopped him,

but I wanted...



   

                   

...revenge.



   

                   

I let him go.



   

                   

I let him get away.



   

                   

He wanted a car.



   

                   

He tried to take Uncle Ben's.



   

                   

Uncle Ben said no...



   

                   

...and then he shot him.



   

                   

Uncle Ben was killed that night...



   

                   

...for being the only one

who did the right thing.



   

                   

And I...



   

                   

I held his hand...



   

                   

...when he died.



   

                   

I've tried to tell you so many times.



   

                   

Just one more little chore.



   

                   

Where are you?



   

                   

- I'm leaving for the night, sir.

- Fine.



   

                   

Your father only obsessed

over his work.



   

                   

Good night, Bernard.



   

                   

Hello, Harry.



   

                   

Otto. What do you want?



   

                   

That precious tritium.



   

                   

But I need more of it this time.



   

                   

More tritium, are you crazy?

You'll destroy the city! You're a hack!



   

                   

Hey! Hey!



   

                   

Stop! Stop! All right!



   

                   

All right, all right, all right.

Put me down. We'll make a deal.



   

                   

Kill Spider-Man,

I'll give you all the tritium you need.



   

                   

On second thought,

bring him to me alive.



   

                   

How do I find him?



   

                   

- Peter Parker.

- Parker?



   

                   

He takes pictures of Spider-Man

for the Bugle.



   

                   

Make him tell you where he is.



   

                   

Have it ready.



   

                   

Don't hurt Peter!



   

                   

Somebody, help!



   

                   

Fire!



   

                   

Alisa, the Chens can't find their

daughter. Have you seen her?



   

                   

Somebody call the fire department!

Help!



   

                   

- Help!

- Is anybody in that building?



   

                   

We think there's a kid

stuck on the second floor.



   

                   

Hey, where you going?!



   

                   

I'm coming!



   

                   

It's okay. It's okay. Come here.

I'm gonna get you out of here.



   

                   

Come on. Come on.



   

                   

Mama.



   

                   

You got some guts, kid.



   

                   

Some poor soul got trapped

on the fourth floor. Never made it out.



   

                   

Oh, man.



   

                   

All right, Billy, let's knock it down

and roll them up. Get on out of here.



   

                   

Let's pick it up! Let's get all this

equipment, let's go! Let's move it!



   

                   

Am I not supposed to have

what I want?



   

                   

What I need?



   

                   

What am I supposed to do?



   

                   

I shouldn't have without...



   

                   

...knocking.



   

                   

Come in.



   

                   

- Hi.

- Hi.



   

                   

Would you like a piece

of chocolate cake?



   

                   

Okay.



   

                   

And a glass of milk?



   

                   

That would be nice.



   

                   

Okay.



   

                   

Thank you.



   

                   

You got a message.



   

                   

It's your aunt.



   

                   

Thanks.



   

                   

What's going on?



   

                   

Oh, they gave me another few weeks,

but I decided the hell with it.



   

                   

I'm moving on.

I found a small apartment.



   

                   

Why didn't you tell me?



   

                   

I'm quite able

to take care of things myself.



   

                   

And Henry Jackson across

the street is giving me a hand...



   

                   

...and I'm giving him $ .



   

                   

That's Henry Jackson?



   

                   

Yes. It's funny what happens

in two years.



   

                   

Nine years old, has great ambitions.



   

                   

Listen, about my last visit...



   

                   

Pish-posh, we needn't talk about it.



   

                   

It's water over the dam or under

the bridge or wherever you like it.



   

                   

But you made a brave move

in telling me the truth.



   

                   

And I'm proud of you.



   

                   

And I thank you, and I...



   

                   

I love you, Peter.



   

                   

So very, very much.



   

                   

Hey. Where are all my comic books?



   

                   

Those dreadful things?

I gave those away.



   

                   

- I put the pans in the box, Mrs. Parker.

- Thank you, Henry.



   

                   

- Hi, Peter.

- Hey, Henry.



   

                   

You're getting tall.



   

                   

Henry, why don't you put those

cookbooks in with the mixer.



   

                   

Okay. You take

Spider-Man's pictures, right?



   

                   

I used to.



   

                   

Where is he?



   

                   

Henry and I agree. We don't see

his picture in the paper anymore.



   

                   

He...



   

                   

...quit.

- Why?



   

                   

Wanted to try other things.



   

                   

He'll be back, right?



   

                   

I don't know.



   

                   

You'll never guess

who he wants to be.



   

                   

Spider-Man.



   

                   

- Why?

- He knows a hero when he sees one.



   

                   

Too few characters out there,

flying around like that...



   

                   

...saving old girls like me.



   

                   

And Lord knows,

kids like Henry need a hero.



   

                   

Courageous, self-sacrificing people...



   

                   

...setting examples for all of us.



   

                   

Everybody loves a hero.



   

                   

People line up for them.

Cheer them. Scream their names.



   

                   

And years later, they'll tell how

they stood in the rain for hours...



   

                   

...just to get a glimpse of the one...



   

                   

...who taught them to hold on

a second longer.



   

                   

I believe there's a hero in all of us...



   

                   

...that keeps us honest...



   

                   

...gives us strength...



   

                   

...makes us noble...



   

                   

...and finally allows us to die

with pride.



   

                   

Even though sometimes

we have to be steady...



   

                   

...and give up the thing

we want the most.



   

                   

Even our dreams.



   

                   

Spider-Man did that for Henry...



   

                   

...and he wonders where he's gone.



   

                   

He needs him.



   

                   

Do you think you could lift that desk

and put it into the garage for me?



   

                   

But don't strain yourself.



   

                   

Okay.



   

                   

Strong focus on what I want.



   

                   

I'm back!



   

                   

I'm back!



   

                   

My back.



   

                   

My back.



   

                   

Honey, you sure you don't want to

invite your friend, the photographer?



   

                   

Peter Parker?



   

                   

Positive.



   

                   

Thought he was your pal.



   

                   

- Peter Parker?

- Yeah.



   

                   

He's just a great big jerk.



   

                   

The world's full of great big jerks.



   

                   

You're adorable.



   

                   

It's just the uniform.



   

                   

Lean your head back for me.



   

                   

- Put my head back?

- Yeah, just do it.



   

                   

Just put my head back?



   

                   

Okay.



   

                   

Wow.



   

                   

I'm back on the moon.



   

                   

You up there with me?



   

                   

- Hiya.

- Surprised?



   

                   

Very.



   

                   

Thanks for coming.



   

                   

Everything okay?



   

                   

You might say so.



   

                   

This is funny.

I'm not sure how to begin.



   

                   

You know how our minds

play tricks on us.



   

                   

- Tell me about it.

- Well, mine did a real number on me.



   

                   

Some part of me heard what you

had to say after the show that night...



   

                   

...that you were different.

But I didn't want to listen.



   

                   

I was afraid to trust you.



   

                   

But I've been thinking about it...



   

                   

Listen, there's more for me to say.



   

                   

I maybe rushed into things.

I thought...



   

                   

Wait a minute.

What are you saying?



   

                   

I'm saying...



   

                   

I thought I could be there for you,

Mary Jane...



   

                   

...but I can't.



   

                   

My mind was playing tricks too.



   

                   

Do you love me, or not?



   

                   

I...



   

                   

...don't.



   

                   

You don't.



   

                   

Kiss me.



   

                   

Kiss you?



   

                   

I need to know something.



   

                   

Just one kiss.



   

                   

What's happening?



   

                   

Peter Parker...



   

                   

...and the girlfriend.



   

                   

What do you want?



   

                   

I want you to find your friend

Spider-Man.



   

                   

Tell him to meet me

at the Westside Tower at  :  .



   

                   

I don't know where he is.



   

                   

Find him.



   

                   

Or I'll peel the flesh off her bones.



   

                   

If you lay one finger on her...



   

                   

You'll do what?



   

                   

Still no word on the whereabouts

of your son's fiancée.



   

                   

Sorry, Jonah.



   

                   

It's all my fault.



   

                   

I drove Spider-Man away.



   

                   

He was the only one who could've

stopped Octavius.



   

                   

Yes.



   

                   

Spider-Man was a hero.



   

                   

I just couldn't see it.



   

                   

He was a...



   

                   

A thief! A criminal!



   

                   

He stole my suit!

He's a menace to the entire city!



   

                   

I want that wall-crawling

arachnid prosecuted!



   

                   

I want him strung up by his web!



   

                   

I want Spider-Man!



   

                   

Where is she?



   

                   

Oh, she'll be just fine. Let's talk.



   

                   

What the hell?



   

                   

You have a train to catch.



   

                   

Help!



   

                   

Help! Help! I can't stop it!

The brakes are gone!



   

                   

- It's Spider-Man.

- Oh, my God, this is it!



   

                   

Tell everyone to hang on!



   

                   

Brace yourselves!



   

                   

Any more bright ideas?



   

                   

I got a few, yeah!



   

                   

Whoa, watch out!



   

                   

We're slowing down.



   

                   

Nice and easy, we'll go down.



   

                   

Slower. Gently.



   

                   

Is he alive?



   

                   

He's...



   

                   

...just a kid.



   

                   

No older than my son.



   

                   

It's all right.



   

                   

We found something.



   

                   

We won't tell nobody.



   

                   

It's good to have you back,

Spider-Man.



   

                   

He's mine!



   

                   

You want to get to him,

you got to go through me.



   

                   

And me.



   

                   

Me too.



   

                   

Very well.



   

                   

Where do you want him?



   

                   

And the tritium?



   

                   

Yes.



   

                   

If only I could cause you the pain

that you've caused me.



   

                   

First we'll see who's behind the mask.



   

                   

I can look into your eyes as you die.



   

                   

Pete.



   

                   

No.



   

                   

It can't be.



   

                   

Harry.



   

                   

Where is she?

Where is he keeping her?



   

                   

- He's got M.J.

- No. All he wanted was the tritium.



   

                   

Tritium?



   

                   

He's making the machine again.



   

                   

When that happens, she'll die,

along with half of New York.



   

                   

- Now, where is he?

- Peter...



   

                   

...you killed my father.



   

                   

There are bigger things

happening here than me and you.



   

                   

Harry, please, I've got to stop him.



   

                   

Hey!



   

                   

Hey!



   

                   

I'm talking to you!



   

                   

You got what you needed for your little

science project, now let me go.



   

                   

I can't let you go,

you'd bring the police.



   

                   

Not that anybody could stop me

now that Spider-Man's dead.



   

                   

He's not dead.



   

                   

- I don't believe you.

- Believe it.



   

                   

Surprise.



   

                   

As soon as you get free...



   

                   

I should have known Osborn

wouldn't have the spine to finish you!



    

                   

Shut it down, Ock. You're going

to hurt a lot more people this time.



    

                   

Well, that's a risk

we're willing to take.



    

                   

Well, I'm not.



    

                   

Hang on, Mary Jane!



    

                   

Run!



    

                   

Let's see you scurry out of this.



    

                   

Now what?



    

                   

Dr. Octavius.



    

                   

We have to shut it down.



    

                   

Please tell me how.



    

                   

Peter Parker?



    

                   

"Brilliant but lazy."



    

                   

Look at what's happening.



    

                   

We must destroy it.



    

                   

I can't destroy it.



    

                   

I won't.



    

                   

You once spoke to me

about intelligence.



    

                   

That it was a gift to be used

for the good of mankind.



    

                   

A privilege.



    

                   

These things have turned you

into something you're not.



    

                   

Don't listen to them.



    

                   

It was my dream.



    

                   

Sometimes, to do what's right,

we have to be steady...



    

                   

...and give up the thing

we want the most.



    

                   

Even our dreams.



    

                   

You're right.



    

                   

He's right.



    

                   

Listen.



    

                   

Listen to me now.



    

                   

Listen to me now.



    

                   

Now, tell me how to stop it.



    

                   

It can't be stopped.



    

                   

It's self-sustaining now.



    

                   

- Think!

- Unless...



    

                   

The river.



    

                   

Drown it.



    

                   

I'll do it.



    

                   

No!



    

                   

Hi.



    

                   

Hi.



    

                   

This is really heavy.



    

                   

M.J.



    

                   

In case we die...



    

                   

You do love me.



    

                   

I do.



    

                   

Even though you said you didn't.



    

                   

I will not die a monster.



    

                   

I think I always knew...



    

                   

...all this time...



    

                   

...who you really were.



    

                   

Then you know

why we can't be together.



    

                   

Spider-Man will always have enemies.



    

                   

I can't let you take that risk.



    

                   

I will...



    

                   

...always be Spider-Man.



    

                   

You and I can never be.



    

                   

Mary Jane!



    

                   

Hello?



    

                   

Who's that?



    

                   

Son.



    

                   

I'm here.



    

                   

Dad?



    

                   

- I thought you were...

- No.



    

                   

I'm alive in you, Harry.



    

                   

Now it's your turn.



    

                   

You swore to make Spider-Man pay.



    

                   

Now make him pay.



    

                   

But Pete's my best friend.



    

                   

And I'm your father.



    

                   

You're weak.



    

                   

You were always weak. You'll always

be weak until you take control.



    

                   

Now you know the truth about Peter.



    

                   

Be strong, Harry.



    

                   

Avenge me.



    

                   

- Avenge me!

- No!



    

                   

Call Deborah.



    

                   

The caterer?



    

                   

Tell her not to open the caviar.



    

                   

Had to do what I had to do.



    

                   

Mary Jane.



    

                   

Peter.



    

                   

I can't survive without you.



    

                   

You shouldn't be here.



    

                   

I know you think we can't be together.



    

                   

But can't you respect me enough

to let me make my own decision?



    

                   

I know there'll be risks.



    

                   

But I want to face them with you.



    

                   

It's wrong that

we should only be half-alive...



    

                   

...half of ourselves.



    

                   

I love you.



    

                   

So here I am,

standing in your doorway.



    

                   

I've always been

standing in your doorway.



    

                   

Isn't it about time somebody

saved your life?



    

                   

Well, say something.



    

                   

Thank you, Mary Jane Watson.



    

                   

Go get them, tiger.









 
Special help by SergeiK
